T_cam_imu:
- [0.021425935450089945, 0.9997678521442828, -0.0022740072327550043, 0.07175407074622919]
- [-0.0029612207141639924, 0.0023379807285593412, 0.9999928824836672, 0.054142809032776545]
- [0.9997660528653538, -0.021419049113322597, 0.003010626697298513, -0.023407668974882546]
- [0.0, 0.0, 0.0, 1.0]


T_lidar_cam:
  - [ 0.00460468,  0.01634899,  0.99985574,  0.036283096029881244]
  - [ -0.99998317, -0.00345452,  0.00466176,  0.05030726551754214]
  - [ 0.00353024, -0.99986038,  0.01633281, -0.01589659548082715]
  - [  0,          0,          0,          1]